Index: common/info.inc
===================================================================
--- common/info.inc	(revision 4)
+++ common/info.inc	(working copy)
@@ -12,8 +12,8 @@
 }
 
 //Allows proper back-button function
-header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
-header("Last-Modified: " . gmdate("D, d M Y H:i:s") . " GMT");
-header("Cache-Control: private, no-store, max-age=0, no-cache, must-revalidate, post-check=0, pre-check=0");
-header("Pragma: no-cache");
+header("Expires: Sat, 01 Jan 2000 00:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+header("Cache-Control: post-check=0, pre-check=0",false);
+session_cache_limiter("must-revalidate");
 ?>
Index: logbook/daily_log/newposts.php
===================================================================
--- logbook/daily_log/newposts.php	(revision 4)
+++ logbook/daily_log/newposts.php	(working copy)
@@ -26,7 +26,7 @@
         $NCTS = True;
     }
     if ($NCTS == True){
-        $post_contents = $post_contents . "<i><br>**This timestamp was manually entered**</i>";
+        $post_contents = $post_contents . "<i><br>**This timestamp was manually entered on " . date('H:i:s, Y-m-d') . "**</i>";
     }
     $timestamp = $big_timestamp ." ". $small_timestamp;
 
@@ -72,5 +72,7 @@
 
 <script>
 var logname = "<?php echo $_GET["log"]; ?>";
-window.location.replace("viewposts.php?log=" + logname);
+var timestamp = "<?php echo $_POST["big_timestamp"]; ?>";
+window.location.replace("viewposts.php?log=" + logname + "&begdate=" + timestamp + "&endate=" + timestamp
+    + "&filters=Calendar");
 </script>
Index: logbook/daily_log/viewposts.php
===================================================================
--- logbook/daily_log/viewposts.php	(revision 4)
+++ logbook/daily_log/viewposts.php	(working copy)
@@ -59,17 +59,11 @@
     "posts_borexino" => "Borexino Logbook"
 );
 
-#Deals with arrows moving forward/back a day. The reason this is a
-#bit more complicated than the arrows on the calendar page is because
-#the month, on the calendar page, is indexed by a variable in the URL
-#while the day on this page is indexed via $_POST variable. Instead of an
-# <a href> picture, as on the calendar page, these arrows must use hidden
-#forms to move from day to day
+#Deals with arrows moving forward/back a day. 
+if (ISSET($_GET["begdate"])){
+    $forward_arrow = date("Y-m-d",strtotime($_GET["begdate"] . "+1 day"));
+    $backward_arrow = date("Y-m-d",strtotime($_GET["begdate"] . "-1 day"));
 
-if (ISSET($_POST["begdate"])){
-    $forward_arrow = date("Y-m-d",strtotime($_POST["begdate"] . "+1 day"));
-    $backward_arrow = date("Y-m-d",strtotime($_POST["begdate"] . "-1 day"));
-
 }
 else{
     $forward_arrow = date("Y-m-d", strtotime(date("Y-m-d") . "+1 day"));
@@ -78,10 +72,11 @@
 
 echo "<center><div id='daylog_title'>";
 #Backward Arrow
-echo "<form method='POST'>";
+echo "<form method='GET'>";
 echo "<input type='hidden' name='filters' value='Calendar'>";
 echo "<input type='hidden' name='begdate' value=$backward_arrow . '00:00:00'>";
 echo "<input type='hidden' name='endate' value=$backward_arrow . '23:59:59'>";
+echo "<input type='hidden' name='log' value=" . $_GET['log'] . ">";
 echo "<input type='image' src='../arrow_left.gif' alt='Submit'>";
 echo "</form>";
 
@@ -88,10 +83,11 @@
 echo "<font size='6.5'> " . $logbook_titles[$_GET["log"]] . " </font>";
 
 #Forward Arrow
-echo "<form method='POST'>";
+echo "<form method='GET'>";
 echo "<input type='hidden' name='filters' value='Calendar'>";
 echo "<input type='hidden' name='begdate' value=$forward_arrow . '00:00:00'>";
 echo "<input type='hidden' name='endate' value=$forward_arrow . '23:59:59'>";
+echo "<input type='hidden' name='log' value=" . $_GET['log'] . ">";
 echo "<input type='image' src='../arrow_right.gif'>";
 echo "</form>";
 echo "</div>";
@@ -99,8 +95,8 @@
 #Generate subtitle based on filters
 echo "<font size='5'>Posts ";
 
-if (!EMPTY($_POST["filters"])){
-    $filters = $_POST["filters"];
+if (!EMPTY($_GET["filters"])){
+    $filters = $_GET["filters"];
 }
 else{
     $filters = "None";
@@ -109,24 +105,24 @@
 if ($filters == "Search"){
     $countdown = 5;
 
-    if (!EMPTY($_POST["begdate"])){
-        echo "after <b>" . $_POST["begdate"] . "</b> ";
+    if (!EMPTY($_GET["begdate"])){
+        echo "after <b>" . $_GET["begdate"] . "</b> ";
         --$countdown;
     }
-    if (!EMPTY($_POST["endate"])){
-        echo "through <b>" . $_POST["endate"] . "</b> ";
+    if (!EMPTY($_GET["endate"])){
+        echo "through <b>" . $_GET["endate"] . "</b> ";
         --$countdown;
     }
-    if (!EMPTY($_POST["post_id"])){
-        echo "with entry #<b>" . $_POST["post_id"] . "</b> ";
+    if (!EMPTY($_GET["post_id"])){
+        echo "with entry #<b>" . $_GET["post_id"] . "</b> ";
         --$countdown;
     }
-    if (!EMPTY($_POST["username"])){
-        echo "written by<b> " . $_POST["username"] . "</b> ";
+    if (!EMPTY($_GET["username"])){
+        echo "written by<b> " . $_GET["username"] . "</b> ";
         --$countdown;
     }
-    if (!EMPTY($_POST["search_string"])){
-        echo "containing<b> " . $_POST["search_string"] . "</b> ";
+    if (!EMPTY($_GET["search_string"])){
+        echo "containing<b> " . $_GET["search_string"] . "</b> ";
         --$countdown;
     }
     if ($countdown == 5){
@@ -137,11 +133,11 @@
 }
 
 elseif ($filters == 'Calendar'){
-    if (strtotime($_POST["begdate"]) == strtotime(date("Y-m-d"))){
+    if (strtotime($_GET["begdate"]) == strtotime(date("Y-m-d"))){
         $date = "Today, " . date("Y-m-d");
     }
     else{
-        $date = date("M. d, Y", strtotime($_POST["begdate"]));
+        $date = date("M. d, Y", strtotime($_GET["begdate"]));
     }
     
     echo "from <b>" . $date . "</b><br></font>";
@@ -161,6 +157,15 @@
 <br>
 <center><a href = "../logbook_frontpage.php?month=0">Back to Calendar</a></center>
 
+<!--Adds "Back to Today" button if date does not match today's date -->
+<?php
+    if (!EMPTY($_GET["begdate"]) & !EMPTY($_GET["endate"])){
+        if ($_GET["begdate"] != date("Y-m-d") & $_GET["endate"] != date("Y-m-d")){ 
+            echo "<center><a href = '../daily_log/viewposts.php?log=" . $_GET["log"] . "'>Back to Today</a></center>";
+        }
+    }
+?>
+
 <!--Actually generates the post tables-->
 <div id="post_display">
 <table>
@@ -204,7 +209,7 @@
 
 
 <tr>
-<td>Non-Current Tiemstamp?  
+<td>Non-Current Timestamp?  
     <input type="text" placeholder="YYYY-MM-DD" name="big_timestamp" size=10 maxlength="10">
     <input type="text" placeholder="HH:MM:SS" name="small_timestamp" size=8 maxlength="8">
 <td>Attach More Files? <button type="button" onclick="additional_uploads()">+</button>
Index: logbook/daily_log/whichposts.php
===================================================================
--- logbook/daily_log/whichposts.php	(revision 4)
+++ logbook/daily_log/whichposts.php	(working copy)
@@ -14,20 +14,22 @@
 $query_string = array("SELECT * FROM " . $_GET["log"]);
 $filter_flag = False;
 for ($i = 0; $i <= 4; ++$i) {
-    if (!EMPTY($_POST[$filter_array[$i][1]]) == True) {
-        if ($filter_flag == False){
-            array_push($query_string, " WHERE ");
-            $filter_array[$i][1] = $_POST[$filter_array[$i][1]];
+    if (!EMPTY($_GET[$filter_array[$i][1]]) == True) {          # If the i'th filter is NOT empty
+        if ($filter_flag == False){                             # If no filters have been registered
+            array_push($query_string, " WHERE ");               # Add this filter to the query
+            $filter_array[$i][1] = $_GET[$filter_array[$i][1]];
             array_push($query_string, implode($filter_array[$i]));
             $filter_flag = True;
         }
-        elseif ($filter_flag == True){
-            $filter_array[$i][1] = $_POST[$filter_array[$i][1]];
+        elseif ($filter_flag == True){  # If a filter has been registered, add other filters with AND
+            $filter_array[$i][1] = $_GET[$filter_array[$i][1]];
             array_push($query_string, " AND " . implode($filter_array[$i]));
         }
     }
 }
-if (EMPTY($_POST["filters"])){
+
+#Use today's posts if there are no filters
+if (EMPTY($_GET["filters"])){
     $filter_array[2][1] = date("Y-m-d");
     $filter_array[3][1] = date("Y-m-d"); 
     array_push($query_string, " WHERE " . implode($filter_array[2]) . " AND " . implode($filter_array[3]));
@@ -65,7 +67,7 @@
         foreach ($filenames as $filename){
             $fileloc = "uploads/" . $_GET["log"] ."/". $filename;
             if (@exif_imagetype($fileloc) != False){
-                echo "<a href = '" .$fileloc. "'> <img src='" .$fileloc. "' height='150' width='300'></a>";
+                echo "<a href = '" .$fileloc. "'> <img src='" .$fileloc. "' height='20%' width='20%'></a>";
             }
             elseif (@exif_imagetype($fileloc) == False){
                 echo "<a href='" .$fileloc. "'>" .$filename. "</a> &nbsp";
Index: logbook/logbook_frontpage.php
===================================================================
--- logbook/logbook_frontpage.php	(revision 4)
+++ logbook/logbook_frontpage.php	(working copy)
@@ -112,7 +112,7 @@
         while ($count = $queries[$j]->fetch(PDO::FETCH_ASSOC)){
             ++$numposts;
         }
-        echo    "<form id='dayform_" .$cur_day."-".$j. "' name='dayform_" .$cur_day."-".$j."' method='POST' action='daily_log/viewposts.php?log=".$chapters[$j]."'>";
+        echo    "<form id='dayform_" .$cur_day."-".$j. "' name='dayform_" .$cur_day."-".$j."' method='GET' action='daily_log/viewposts.php?log=".$chapters[$j]."'>";
         echo    "<input type='hidden' name='begdate' value='" . $cal_date1 . "'>";
         echo    "<input type='hidden' name='log' value='".$chapters[$j]."'>";
         echo    "<input type='hidden' name='endate' value='" . $cal_date2 . "'>";
Index: logbook/search.php
===================================================================
--- logbook/search.php	(revision 4)
+++ logbook/search.php	(working copy)
@@ -1,13 +1,13 @@
 <html>
 <body>
 <?php
-$log = $_POST["log"];
-$post_id = $_POST["post_id"];
-$username = $_POST["username"];
-$begdate = $_POST["begdate"];
-$endate = $_POST["endate"];
-$search_string = $_POST["search_string"];
-$filters = $_POST["filters"];
+$log = $_GET["log"];
+$post_id = $_GET["post_id"];
+$username = $_GET["username"];
+$begdate = $_GET["begdate"];
+$endate = $_GET["endate"];
+$search_string = $_GET["search_string"];
+$filters = $_GET["filters"];
 
 if (is_numeric($post_id) == False AND !EMPTY($post_id)){
     $good_to_go = False;
@@ -21,7 +21,7 @@
 }
 ?>
 
-<form id="inbetween" name="inbetween" method = "POST" action=<?php echo "/pocarlab/logbook/daily_log/viewposts.php?log=" . $_POST["log"]; ?>>
+<form id="inbetween" name="inbetween" method = "GET" action=<?php echo "/pocarlab/logbook/daily_log/viewposts.php?log=" . $_GET["log"]; ?>>
 <input type="hidden" name="username" value ="<?php echo $username; ?>">
 <input type="hidden" name="post_id" value = "<?php echo $post_id; ?>">
 <input type="hidden" name="begdate" value = "<?php echo $begdate; ?>">
@@ -28,6 +28,7 @@
 <input type="hidden" name="endate" value = "<?php echo $endate; ?>">
 <input type="hidden" name="search_string" value = "<?php echo $search_string; ?>">
 <input type="hidden" name="filters" value = "<?php echo $filters; ?>">
+<input type="hidden" name="log" value = "<?php echo $log; ?>">
 </form>
 </body>
 
Index: logbook/search_filters.php
===================================================================
--- logbook/search_filters.php	(revision 4)
+++ logbook/search_filters.php	(working copy)
@@ -1,5 +1,5 @@
 <div id = "post_form">
-<form id="search_filters" name="search_filters" method="POST" action="/pocarlab/logbook/search.php">
+<form id="search_filters" name="search_filters" method="GET" action="/pocarlab/logbook/search.php">
 
 <center>
 <table>
